getting_started/step_by_step/your_first_game.rst

```diff
@@ -629,6 +629,7 @@ choose one of the three animation types:
 
     public override void _Ready()
     {
+        var _mobType = GetNode<AnimatedSprite>("AnimatedSprite").Frames.GetAnimationNames();
```

Comment:
This should be `_mobTypes` to match with the next line.

Reply:
You're right!
I don't use copy and paste enough.